src/main.js

```diff
@@ -135,52 +122,53 @@ showMyPosterButton.addEventListener('click', showMyPoster);
 savePosterButton.addEventListener('click', savePoster);
 showSavedButton.addEventListener('click', displaySaved);
 backToMainButton.addEventListener('click', savedToggle);
-
-// functions and event handlers go here 👇
-function getRandomIndex(array) {
-  return Math.floor(Math.random() * array.length);
-}
-
+collectionGrid.addEventListener('dblclick', deleteSavedPoster);
 function toggleHidden(element) {
-  element.classList.toggle("hidden")
+  element.classList.toggle("hidden");
 }
-
 function formToggle() {
-    toggleHidden(posterForm);
-    toggleHidden(mainPoster);
+  toggleHidden(posterForm);
+  toggleHidden(mainPoster);
 }
-
 function savedToggle() {
   toggleHidden(mainPoster);
   toggleHidden(postersCollection);
 }
-
+function getRandomIndex(array) {
+  return Math.floor(Math.random() * array.length);
+}
+function buildNewDisplay(poster) {
+  image.src = poster.imageURL;
+  title.innerText = poster.title;
+  quote.innerText = poster.quote;
+}
+function displayPoster() {
```

Comment:
I would consider renaming this function. Is it displaying a _selected_ poster, or generating a random one?